src/base/QXmppRosterIq.h

```diff
@@ -86,6 +93,9 @@ class QXMPP_EXPORT QXmppRosterIq : public QXmppIq
         // can be subscribe/unsubscribe (attribute "ask")
         QString m_subscriptionStatus;
         QSet<QString> m_groups;
+        // XEP-0405: Mediated Information eXchange (MIX): Participant Server Requirements
+        bool m_isMixChannel = false;
```

Comment:
This breaks ABI compatibility. If we're going to do this let's at least future-proof it by moving everything to a d-pointer.

Reply:
Useful link about d-pointer and about ABI compatibility in common: 
https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B

And I would prefer the separate commits for this task. For not mixing this with changes related to providing of new features in this PR...

Reply:
done